src/Reflection/Adapter/ReflectionFunction.php

```diff
@@ -27,7 +27,7 @@ public function __construct(BetterReflectionFunction $betterReflectionFunction)
      */
     public static function export($name, $return = null)
     {
-        BetterReflectionFunction::export(...func_get_args());
+        return BetterReflectionFunction::export(...func_get_args());
```

Comment:
Hmm, I know there's been a couple of comments on these, but `::export` methods really do return nothing:

https://3v4l.org/uLmk8

There's certainly a discrepancy here (phpdocs vs reality), so which do we pick? What the docs/types say, or how it actually behaves? So far we've been obeying the real-world behaviour (which I'd favour, but is of course incompatible with static analysis...)

/cc @Ocramius

Reply:
Reality would be better overall.

Reply:
I'm confused. It *can* return e.g. https://3v4l.org/oolCP